vet-service/src/main/java/com/petclinic/vet/presentationlayer/VetController.java

```diff
@@ -23,11 +22,15 @@
 @RequestMapping("vets")
 public class VetController {
 
-    @Autowired
-    VetService vetService;
+    private final VetService vetService;
```

Comment:
[Nitpick]: add @RestController and @RequiredArgsConstructor instead of making a constructor.